interni/VIN_20230620/contenuti/svolgimento.tex

```diff
@@ -0,0 +1,11 @@
+\section{Svolgimento}
+
+È stato definito quale tool utilizzare per generare la documentazione Angular.
+
+Compodoc fornisce anche il calcolo del code coverage su un progetto Angular:
+
+\includegraphics[width = 0.9\linewidth]{img/compodoc.png}
+
+Angular è stato scelto per la realizzazione della WebApp lato front-end, pertanto si è resa necessaria la ricerca di un tool per generare la documentazione relativa a quanto prodotto, necessaria ai fini del progetto.
```

Comment:
Ok, ma che Requisiti copre?

Reply:
Questo non lo so sinceramente

Reply:
Va bene, ma scrivi il requisito nello specifico prendendo il codice dall'analisi dei requisiti.

Se nell'analisi dei requisiti manda, domandati perché ed in caso apri la issue per sistemare la cosa in AdR

Reply:
Giorgio ha detto che torna Mercoledì dopo l'esame di IA, quindi ho cercato io nel documento dell'AdR. I requisiti che vengono coperti dalla documentazione direi che sono presenti alla voce "requisiti di qualità", più precisamente l'RQ_01.

Reply:
RQ_01 Obbligatorio Il progetto deve includere un manuale utente che fornisca spiegazione dettagliate sulle funzionalità dell’applicativo
Capitolato
RQ_02 Obbligatorio Il progetto deve includere un manuale per sviluppatori che fornisca spiegazione dettagliate su come svolgere manutenzione sull’applicativo
Capitolato

Ti direi che è più corretto RQ_2

Reply:
La documentazione è richiesta come da requisito di qualità RQ_02 definito nel documento "Analisi dei Requisiti", il quale prevede che il progetto includa un manuale per sviluppatori che fornisca spiegazione dettagliate su come svolgere manutenzione sull’applicativo. Questo requisito, come definito dal capitolato, va rispettato obbligatoriamente e diventa quindi imprescindibile la presenza del manuale per gli sviluppatori.

Reply:
La documentazione è richiesta come da requisito di qualità RQ_02 definito nel documento "Analisi dei Requisiti", il quale prevede che il progetto includa un manuale per sviluppatori che fornisca spiegazione dettagliate su come svolgere manutenzione sull’applicativo. Questo requisito, come definito dal capitolato, va rispettato obbligatoriamente.

Reply:
```suggestion 
La documentazione è richiesta come da requisito di qualità RQ_02 definito nel documento "Analisi dei Requisiti", il quale prevede che il progetto includa un manuale per sviluppatori che fornisca spiegazione dettagliate su come svolgere manutenzione sull'applicativo. Questo requisito, come definito dal capitolato, va rispettato obbligatoriamente.
```